google-cloud-contrib/google-cloud-nio/pom.xml

```diff
@@ -112,6 +112,18 @@
               <shadedPattern>shaded.cloud-nio.org</shadedPattern>
             </relocation>
             <relocation>
+              <pattern>io</pattern>
```

Comment:
I'm honestly not familiar enough with how the shading plugin works. But this will probably be fine.